tests/dynamic_data/publications/crs/vectors/__init__.py

```diff
@@ -270,6 +270,12 @@ def generate(workspace=None):
                                                                        'precision': bboxes['CRS:84']['precision'],
                                                                        }),
                     *wms_bbox_actions,
+                    Action(publication.internal.point_coordinates, {'point_id': 1,
```

Comment:
Nice. I would like to assert transformation of all items in EXP_POINT_COORDINATES.